docs/changelog.rst

```diff
@@ -39,6 +39,9 @@ Bug fixes:
 * :doc:`/plugins/lyrics`: Fix crash bug when beautifulsoup4 is not installed.
   :bug:`4027`
 
+* :doc:`/plugins/discogs`: Fix regex for new style of regex.
```

Comment:
Nitpick:
```suggestion
* :doc:`/plugins/discogs`: Adapt regex to new URL format.
```
seems clearer to me

Reply:
I agree with the wording change, updated appropriately